qa/L0_s3_swiftstack/infer_test.py

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
```

Comment:
L0_storage_swiftstack to match naming convention

Reply:
Fixed